### DISCUSSION TITLE
List of planned features.

### DISCUSSION THREAD

**Discussion by chirag-droid** [chirag-droid/animeflix#84] [Ideas]
Here are some features which I am looking forward to add.

- [x] The site logo. The logo will be placed at top left corner and can be used to go to homepage.
- [x] Genre page. 
- [x] Let the user choose, if they want to proxy videos or not
- [x] Keyboard shortcuts.
- [x] List of episodes given at watch page.
- [x] A option to stream dubbed anime, if available.
- [ ] Studios page to show animes from a particular studio.
- [ ] Search page to refresh and show more than just 50 results.
- [x] Watch history.
- [ ] And lastly connect to your existing anilist account.

**Comment by Kadantte** [comment]
Hi, the website needs some tweaks.
on the title bar it needs to show the proper title if we are at Home it should show Aniflex and when we open an Anime it shows it's name Ex: BLEACH

![image](https://user-images.githubusercontent.com/11579313/166503731-12806094-e270-4ff2-9a43-79f352a7b7d6.png)

**Reply by chirag-droid** [reply]
This has been fixed now.

Now there is better seo and if you share site link somewhere it will show embed too.

**Comment by acekun** [comment]
great work. I'm running this behind a reverse proxy to access from work :) 
really need this "List of episodes given at watch page."

**Reply by chirag-droid** [reply]
I am glad. also I have finished working on that feature just a little left.

My exams are going on so didn't get much time today

**Reply by chirag-droid** [reply]
done!
